controllers/bfdprofile_controller_test.go

```diff
@@ -110,6 +110,55 @@ var _ = Describe("BFD Controller", func() {
 		})
 	})
 
+	Context("Creating invalid BFDProfiles", func() {
+		AfterEach(func() {
+			err := cleanTestNamespace()
+			Expect(err).ToNot(HaveOccurred())
+		})
+		By("Creating new profile with invalid detect multiplier value (over maximum limit)")
```

Comment:
nit: add a line above (same for all the others By / It)

Reply:
fixed. will make sure to follow that rule next time